web/src/components/nav/betaNotice.js

```diff
@@ -0,0 +1,20 @@
+import { Route } from "react-router-dom";
+import { DismissableMessage } from "../ui";
+
+const MESSAGESTYLE = {
+	background: "#fec61e",
+	marginTop: "1em",
+	display: "inline-block",
+	width: "100%",
+};
```

Comment:
If we want to follow how the other files are structured, this should be moved to its own css file.

Reply:
I would agree. Perhaps we have a `betaNotice.css` file and the `DismissableMessage` be passed the class?

Reply:
Sounds good.